index.html

```diff
@@ -67,24 +67,42 @@ <h1>Hello, world!</h1>
         <div class="col-md-4">
           <h2>Heading</h2>
           <p>Donec id elit non mi porta gravida at eget metus. Fusce dapibus, tellus ac cursus commodo, tortor mauris condimentum nibh, ut fermentum massa justo sit amet risus. Etiam porta sem malesuada magna mollis euismod. Donec sed odio dui. </p>
-          <p><a class="btn btn-default" href="#" role="button">View details &raquo;</a></p>
+          <p>
+            <!-- jQuery Like Button -->            
+            <a class="like btn btn-primary" role="button">
+              <span class="glyphicon glyphicon-thumbs-up"></span>
+              <input class="qty1" name="qty1" type="text" readonly="readonly" value="0"/>
+            </a>              
+            <a class="btn btn-default" href="#" role="button">View details &raquo;</a></p>
         </div>
         <div class="col-md-4">
           <h2>Heading</h2>
           <p>Donec id elit non mi porta gravida at eget metus. Fusce dapibus, tellus ac cursus commodo, tortor mauris condimentum nibh, ut fermentum massa justo sit amet risus. Etiam porta sem malesuada magna mollis euismod. Donec sed odio dui. </p>
-          <p><a class="btn btn-default" href="#" role="button">View details &raquo;</a></p>
+          <p>
+            <!-- jQuery Like Button -->            
+            <a class="like btn btn-primary" role="button">
+              <span class="glyphicon glyphicon-thumbs-up"></span>
+              <input class="qty1" name="qty1" type="text" readonly="readonly" value="0"/>
+            </a>            
+            <a class="btn btn-default" href="#" role="button">View details &raquo;</a></p>
        </div>
         <div class="col-md-4">
           <h2>Heading</h2>
           <p>Donec sed odio dui. Cras justo odio, dapibus ac facilisis in, egestas eget quam. Vestibulum id ligula porta felis euismod semper. Fusce dapibus, tellus ac cursus commodo, tortor mauris condimentum nibh, ut fermentum massa justo sit amet risus.</p>
-          <p><a class="btn btn-default" href="#" role="button">View details &raquo;</a></p>
+          <p>
+            <!-- JavaScript Like Button -->
+            <a class="btn btn-primary" role="button" onclick="buttonClick()">
+              <span class="glyphicon glyphicon-thumbs-up"></span>
+              <input id="clicks" type="text" readonly="readonly" value="0" class="counter"></input>
+            </a>            
+            <a class="btn btn-default" href="#" role="button">View details &raquo;</a></p>
         </div>
       </div>
 
       <hr>
 
       <footer>
-        <p>&copy; Company 2015</p>
+        <p>&copy; Nifty-Club 2019</p>
```

Comment:
I agree with changing the footer to Nifty-Club 2019.

Reply:
Great eye!

Reply:
I think we should add a "!" at the end of "Nifty-Club 2019" so it's "Nifty-Club 2019!"

Reply:
Other than that change, I say ship it!